src/plans/D3MAavePlan.sol

```diff
@@ -116,26 +116,21 @@ contract D3MAavePlan is ID3MPlan {
         else revert("D3MAavePlan/file-unrecognized-param");
         emit File(what, data);
     }
-
     function file(bytes32 what, address data) external auth {
         if (what == "tack") tack = InterestRateStrategyLike(data);
         else revert("D3MAavePlan/file-unrecognized-param");
         emit File(what, data);
     }
 
-    function _maxBar() internal view returns (uint256) {
-        return tack.getMaxVariableBorrowRate();
-    }
-
-    // --- Automated Rate targeting ---
```

Comment:
we should keep the comment here

Reply:
Didn't think it adds anything (it is a relic from a version far back).
Anyway returned it back in to streamline the discussion.